benchmarks/distribution.py

```diff
@@ -7,8 +7,10 @@
 import probnum.prob as prob
 import probnum.linalg.linops as linops
 
+from benchmarks.benchmark_utils import RANDOM_5x5_SPD_MATRIX
+
 # Module level variables
-distribution_names = [
+DISTRIBUTION_NAMES = [
```

Comment:
Once you merge the most recent state of ProbNum, this should already be fixed, but these are `RANDVAR_NAMES` now with the refactoring done in #164 .